.evergreen.yml

```diff
@@ -234,6 +234,20 @@ functions:
       content_type: text/html
       permissions: public-read
       display_name: storybook-
+  
+  attach-codegen-diff:
+    command: s3.put
+    type: system
+    params:
+      aws_key: ${aws_key}
+      aws_secret: ${aws_secret}
+      local_files_include_filter:
+        - "spruce/bin/codegen.diff"
+      remote_file: spruce/${task_id}/codegen/
+      bucket: mciuploads
+      content_type: text/plain
+      permissions: public-read
+      display_name: "codegen diff"
```

Comment:
Is there any way we can get this to say `codegen diff` only?

<img width="757" alt="Screen Shot 2022-07-20 at 2 56 59 PM" src="https://user-images.githubusercontent.com/47064971/180068757-53406fac-c088-4a9f-81b7-19b0608fd26c.png">

Reply:
I could remove the display name so it would only say `codegen.diff`

Reply:
done